src/services/loadflow.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { backendFetchJson } from '../utils/rest-api';
+
+const PREFIX_LOADFLOW_SERVER_QUERIES = `${process.env.REACT_APP_API_GATEWAY}/loadflow`;
+
+export function getLoadFlowUrl() {
+    return PREFIX_LOADFLOW_SERVER_QUERIES + '/v1/';
```

Comment:
make '/v1' a const and exported

Reply:
Done, thank you
for v1 in rest-api, I will fix that later with next PRs.